services/data-info/src/data_info/routes/stats.clj

```diff
@@ -9,6 +9,8 @@
 
 (defroutes* stat-gatherer
 
+            ; FIXME Update metadactyl exception handling when data-info excptn hndlg updated
+            ; metadactyl catches exceptions thrown from this EP.
```

Comment:
Yeah, I'm not sure about Donkey requiring an update, since Donkey should just be a pass-through and forward errors straight back to the client. Unless we need to update Donkey so that it doesn't catch data-info errors and log them again?